spring-cloud-starter-dataflow-server-local/src/test/java/org/springframework/cloud/dataflow/server/local/TestUtils.java

```diff
@@ -95,4 +99,14 @@ public static void setField(String name, Object target, Object value) throws Exc
 		return (T) ReflectionUtils.invokeMethod(method, target, args);
 	}
 
+	public static Map<String, String> toImmutableMap(
+			String... entries) {
+		if(entries.length % 2 == 1)
+			throw new IllegalArgumentException("Invalid entries");
```

Comment:
Exception comment should say.  Entries should contain even number of values.   

Will fix on merge.